### PR TITLE
Change power.schedule to TEXT

### PR DESCRIPTION
**Pull request by tuxudo**

### CODE REVIEW

File: app/modules/power/migrations/004_power_schedule_text.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                    $this->exec($sql);
+                
+                    // Delete old table
+                    $sql = 'DROP TABLE power_tmp';
+                    $this->exec($sql);
+
```
Comment (bochoven) [inline app/modules/power/migrations/004_power_schedule_text.php]: You need to add indexes

Comment (tuxudo) [inline app/modules/power/migrations/004_power_schedule_text.php]: I'm not sure what the proper method is for adding indexes via a migration.

Comment (bochoven) [inline app/modules/power/migrations/004_power_schedule_text.php]: Check https://github.com/munkireport/munkireport-php/blob/master/app/modules/machine/migrations/002_machine_add_indexes.php

File: app/modules/power/migrations/004_power_schedule_text.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                    $sql = 'DROP TABLE power_tmp';
+                    $this->exec($sql);
+                
+                // Get database handle
+                $dbh = $this->getdbh();
+
+                $dbh->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+
+                // Wrap in transaction
+                $dbh->beginTransaction();
```
Comment (bochoven) [inline app/modules/power/migrations/004_power_schedule_text.php]: Please move beginTransaction to the start of the sqlite section. Transactions make the migration atomic.

**Pull request merged**